src/docs/Components/Form/PhoneNumberInput/accessibility.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: 'Accessibility'
+order: 3
+---
+
+## Accessibility and semantic
+<HintItem>
+The button element (mc-phone-number__button) must always have an id, a aria-haspopup and aria-labelledby attribute.
+</HintItem>
+<HintItem>
+The list element (mc-phone-number__list) must always have a role list or listbox
```

Comment:
The list element (mc-phone-number__list) must always have a <strong>role="list"</strong> or <strong>role="listbox"</strong> attribute.